### ISSUE TITLE
SKEW Profile causes filament cutter to "Miss"

### ISSUE THREAD

**Issue by arnoldstrife** [moggieuk/Happy-Hare#237]
Having a SKEW profile (https://www.klipper3d.org/Skew_Correction.html) can cause the toolhead to miss the filament cutter. I get

Reason: Unload sequence failed: Error running _MMU_CUT_TIP: Move out of range: -1.854 355.000 2.817 [5.000]

Because with the skew correction it's trying to move the toolhead to a negative x axis. I fixed it myself by adding

SET_SKEW CLEAR=1
to the beginning of the gcode macro and
SKEW_PROFILE LOAD=CaliFlower //Califlower is the name of my skew profile
at the end.
Probably should have something built in to fix it.

**Comment by moggieuk** [comment]
Yes, this is true. The solution is to move the location of the cutter slightly away from the edge.  It is always risky to move/print right at the limit with skew correction on because even a 0.00001 mm movement causing an overhang in the print volume will cause klipper to stop.  This is how klipper is designed and the purpose of a the fixed build volume.
